selfdrive/test/test_car_models.py

```diff
@@ -433,6 +433,18 @@ def get_route_log(route_name):
     'carFingerprint': MAZDA.CX5,
     'enableCamera': True,
   },
+  "10b5a4b380434151|2020-08-26--17-11-45": {
+    'carFingerprint': MAZDA.CX9,
+    'enableCamera': True,
+  },
+  "74f1038827005090|2020-08-27--10-23-22": {
+    'carFingerprint': MAZDA.Mazda3,
+    'enableCamera': True,
+  },
+  "0f702597446474bf|2020-08-12--08-35-04": {
```

Comment:
neither is this one

Reply:
How do I know if they are uploaded? These routes show up on useradmin page.

Reply:
all the rlogs and fcamera need to be uploaded too